### ISSUE TITLE
I can't simulate a spam result

### ISSUE THREAD

**Issue by alehandrof** [edalzell/statamic-akismet#2]
I believe I've set up Akismet correctly, but I can't catch any spam. Here's a form submission that doesn't get caught as spam:

```
contact__name: viagra-test-123
contact__email: akismet-guaranteed-spam@example.com
contact__message: 'Test, test, test.'
```

Both the name and email use the triggers that [Akismet's API docs](https://akismet.com/development/api/#detailed-docs) state will simulate a spam result.

Can you please confirm whether this submission works on your end?

**Comment by edalzell** [comment]
Can you please share the addon settings and the template form code?

**Comment by edalzell** [comment]
<img width="753" alt="2018-08-27 at 7 33 pm" src="https://user-images.githubusercontent.com/6069653/44697407-756c8b00-aa30-11e8-85f5-e4d9fc23a33f.png">

**Comment by alehandrof** [comment]
Hmm. I suspect that Akismet might be ignoring my development domain. I'll come back to you if the problem persists on my staging server. Feel free to close the issue in the meantime.

**Comment by alehandrof** [comment]
### Addon settings

```
akismet_key: %% key %%
roles:
  - 6414c825-9b31-4ec6-a49a-8ef8c5321621
forms:
  -
    form_and_fields:
      form: profile-email
      author_field: contact__name
      email_field: contact__email
      content_field: contact__message
  -
    form_and_fields:
      form: profile-email
      author_field: contact__name
      email_field: contact__email
      content_field: contact__message
```

### Contact fieldset

```
title: Contact
honeypot: sweetraven
fields:
  contact__name:
    display: Name
    validate: required
  contact__email:
    display: Email
    validate: required|email
  contact__message:
    display: Message
    validate: required
columns:
  - contact__name
  - contact__email
  - contact__message
email:
  -
    to: %% Organizational email %%
    from: '{{ contact__email }}'
    subject: "Message from {{ contact__name }}"
    template: contact
```

The form template itself is pretty complex. Are you sure you need it?

**Comment by edalzell** [comment]
Is the form submitted? i.e. you get a submission?

**Comment by alehandrof** [comment]
Yes, I do get a submission.

**Comment by alehandrof** [comment]
OK, I figured it out. It was something silly on my end.

When I clicked on "Spam Queue", I thought it was showing me that it hadn't caught any spam. But it was only doing so for the first form. If clicked on the other form, I was able to see the caught spam:

![capto_capture 2018-08-31_14-18-52_](https://user-images.githubusercontent.com/1588449/44909670-f4134200-ad28-11e8-913f-a566c46a4107.png)

Sorry for the false alarm. Is it possible to add a badge for unseen spam emails from all accounts on the "Spam Queue", much like the "Updater" above it?
